tests/queries/0_stateless/02174_cte_scalar_cache.sql

```diff
@@ -18,6 +18,16 @@ SELECT '02177_CTE_GLOBAL_OFF', a5 FROM system.numbers LIMIT 100
         FORMAT Null
 SETTINGS enable_global_with_statement = 0;
 
+WITH
```

Comment:
This test looks strange, why the result is different with `allow_experimental_analyzer` from `02177_CTE_GLOBAL_ON` or `02177_CTE_GLOBAL_OFF`.

Reply:
It should be same as 02177_CTE_GLOBAL_ON, the only difference with alias are not considered as same queries, thus we have one more cache miss.